test/cppsuite/test_harness/workload/workload_validation.h

```diff
@@ -200,25 +192,17 @@ class workload_validation {
              * the key has been inserted previously in an existing collection and can be safely
              * deleted.
              */
-            database.collections.at(collection_name).keys.at(key).exists = false;
-            delete database.collections.at(collection_name).values;
```

Comment:
I think this was previously clearing every value for the collection rather than just for the key, so I've changed `delete_record` to erase only the key we're deleting.

I'm assuming this was a bug but wanted to double-check.

Reply:
Seems that it was a bug, thanks for fixing it.